spec/helpers/basespace_helper_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+require "webmock/rspec"
+
+RSpec.describe BasespaceHelper, type: :helper do
```

Comment:
They are tested in `basespace_controller_spec.rb`, since they are exposed via endpoints. The methods tested here aren't exposed via controller endpoints.